ZenPacks/zenoss/WSMAN/tests/testWSMANDataSource.py

```diff
@@ -0,0 +1,78 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2018, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+from twisted.python.failure import Failure
+
+from Products.ZenEvents import ZenEventClasses
+from Products.ZenTestCase.BaseTestCase import BaseTestCase
+
+from ZenPacks.zenoss.WSMAN.datasources.WSMANDataSource import \
+    WSMANDataSourcePlugin
+from ZenPacks.zenoss.WSMAN.tests.utils import load_pickle
+
+
+class TestWSMANDataSourcePlugin(BaseTestCase):
+
+    def setUp(self):
+        self.plugin = WSMANDataSourcePlugin()
+        self.results = load_pickle('success')
+        self.config = load_pickle('config')
+
+    def test_onSuccess(self):
+        data = self.plugin.onSuccess(self.results, self.config)
+        event = data['events'][0]
+
+        self.assertEquals(len(data['values']), 6)
+        self.assertEquals(len(data['events']), 1)
+        self.assertEquals(event['severity'], ZenEventClasses.Clear)
+        self.assertEquals(event['eventClass'], '/Status/Events')
+
+    def test_onError(self):
+        f = Failure(Exception('test_onError'))
+
+        data = self.plugin.onError(f, self.config)
+        event = data['events'][0]
+
+        self.assertEquals(len(data['values']), 0)
+        self.assertEquals(len(data['events']), 1)
+        self.assertEquals(event['severity'], ZenEventClasses.Error)
+        self.assertEquals(event['eventClass'], '/Status/Events')
+
+    def test_onSuccess_event_class_defined(self):
+        # define event class
```

Comment:
The comment doesn't bring any value. FYI https://refactoring.guru/smells/comments